parallel-install/pkg/git/clone_test.go

```diff
@@ -62,3 +62,11 @@ func TestCloneRepo(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "Add README\n", commit.Message)
 }
+
+// func TestRealCloneRepo(t *testing.T) {
```

Comment:
Please either uncomment or delete this code.

Reply:
This is not a unit test, since it actually clones the repo. But I kept it to easily check bugs at the boundary to go-git later (as we had when trying to checkout specific versions). Should I still delete it?

Reply:
Yes, with modern version control systems leaving commented stuff for casual usage is not necessary.
You can always explore git revisions and copy this snippet when you need it.